01_04_Damage_Types_and_Armor_Types.txt

```diff
@@ -21,6 +23,24 @@ shielding. Lasers deal full damage to armored targets; If a player wearing
 ballistic armor is hit with a laser, they receive damage as if they weren't 
 wearing any armor.
 
+== Fall Damage ==
+
+    Beyond weapon damage, there are also environmental threats to consider. An 
+important note is on fall damage. A player can fall up to their Dex/2 in 
```

Comment:
Non-blocking: This wording is a little weird here. Something like:
Weapons are not the only danger in the universe; falling can be just as deadly.
Basically, I think the two sentences can be combined into something that flows more.